plugins/woocommerce/includes/abstracts/abstract-wc-product.php

```diff
@@ -1947,6 +1947,37 @@ public function get_price_html( $deprecated = '' ) {
 		return apply_filters( 'woocommerce_get_price_html', $price, $this );
 	}
 
+	/**
+	 * Returns the Cost of Goods Sold value in html format.
+	 *
+	 * @return string
+	 */
+	public function get_cogs_value_html() {
+		$value = $this->get_cogs_total_value();
+		$html  = 0.0 === $value ?
+			/**
+			 * Filter to customize how an empty Cost of Goods Sold value for a product gets rendered to HTML.
+			 *
+			 * @param string $html The rendered HTML.
+			 * @param WC_Product $product The product for which the cost is rendered.
+			 *
+			 * @since 9.7.0
+			 */
+			apply_filters( 'woocommerce_empty_cogs_html', '', $this ) :
+			wc_price( $value ) . $this->get_price_suffix();
+
+		/**
+		 * Filter to customize how the Cost of Goods Sold value for a product gets rendered to HTML.
+		 *
+		 * @param string $html The rendered HTML.
+		 * @param float $value The cost value that is being rendered.
+		 * @param WC_Product $product The product for which the cost is rendered.
+		 *
+		 * @since 9.7.0
```

Comment:
Noting that these will presumably need to get updated to 9.8.0.

```suggestion
		 * @since 9.8.0
```